architecture.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Justin Hernandez</title>
+    <link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+
+  </head>
+
+  <body>
+    <header>
+        <div class="logo">
+          <h1><a href="#">Justin Hernandez</a></h1>
+        </div> <!-- End Logo -->
+        <div class="main__nav">
+          <ul>
+            <li class="listorder--float" id="showSecondaryNav__Design">
+              <a href="#">Design Work
+              <img src="images/arrow_down.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float hide" id="hideSecondaryNav__Design">
+              <a href="#">Design Work<img src="images/arrow_up.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float" id="showSecondaryNav__Photography">
+              <a href="#">Photography
+              <img src="images/arrow_down.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float hide" id="hideSecondaryNav__Photography">
+              <a href="#">Photography<img src="images/arrow_up.svg" class="arrow"></a>
+            </li>
+
+            <li class="listorder--float"><a href="#">Personal Project</a></li>
+            <li class="listorder--float"><a href="#">About Me</a></li>
+
+          </ul>
+
+        </div> <!-- End Main Nav -->
+        <div class="socialmedia">
+          <ul>
+            <li class="listorder--float"><a href="#"><img src="http//placehold.it/75x75"></a></li>
```

Comment:
Start filling these - there are some fantastic free SVGs of all the common social media icons.